src/yunohost/app.py

```diff
@@ -148,8 +148,12 @@ def app_list(offset=None, limit=None, filter=None, raw=False):
 
     for applist in applists:
         if '.json' in applist:
+            list_name=applist.split('.json', 1)[0]
             with open(repo_path +'/'+ applist) as json_list:
-                app_dict.update(json.loads(str(json_list.read())))
+                list_content=json.loads(str(json_list.read()))
+                for k in list_content.keys():
+                    list_content[k]['store']=list_name
```

Comment:
What's the goal of this key? It seems not to be related to the other part, isn't it?